ops/testing.py

```diff
@@ -1191,6 +1190,196 @@ def _get_backend_calls(self, reset: bool = True) -> List[Tuple[Any, ...]]:
             self._backend._calls.clear()
         return calls
 
+    def add_model_secret(self, app_or_unit: AppUnitOrName, content: Dict[str, str]) -> str:
```

Comment:
It's not a problem as such, and the previous iteration of this PR had it named `add_secret`, but we had a comment explaining why this was not the same as `Application.add_secret` (implying "be careful"), so I think the distinct name is best just to avoid confusion.